pkgs/applications/emulators/dosbox-staging/default.nix

```diff
@@ -20,22 +25,17 @@
 , ninja
 , opusfile
 , pkg-config
-, irr1
-, SDL2
-, SDL2_image
-, SDL2_net
 , speexdsp
-, stdenv
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (self: {
   pname = "dosbox-staging";
   version = "0.80.1";
 
   src = fetchFromGitHub {
-    owner = pname;
-    repo = pname;
-    rev = "v${version}";
+    owner = "dosbox-staging";
+    repo = "dosbox-staging";
```

Comment:
why not use `pname`?

Reply:
Because it will never change. It makes few to no sense to parameterize `pname` all around.

Reply:
I think it's more about readability; folks expect `pname` and it causes (even slight) some mental overhead to read something different

Reply:
I have encontered parties on both sides. Many believe that indeed pname all around confounds more than helps.

Given that there are many examples where `pname != src.repo` already, I believe I am justified to not parameterize `pname` all around, especially when defining sources.

Reply:
Indeed, by running `rg pname` in Nixpkgs, I have found 4527 occurrences of `repo = pname` against 33902 occurrences of `repo = "XXXX"`. This is appoximately 1/7.

So I don't believe readability arguments apply here.